Source/DafnyCore/Options/VerifyIncludesOption.cs

```diff
@@ -0,0 +1,13 @@
+namespace Microsoft.Dafny; 
+
+public class VerifyIncludesOption : BooleanOption {
+  public static readonly VerifyIncludesOption Instance = new();
+  public override string LongName => "verify-includes";
```

Comment:
Name feels a little off to me, like we're verifying the include statements themselves rather than the included content. How about `verify-included-files`?